src/PiTop.nuget/PiTop.nuget.csproj

```diff
@@ -5,17 +5,17 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <PackageVersion>0.0.1</PackageVersion>
-    <PackageId>piTop</PackageId>
+    <PackageId>pi-top</PackageId>
     <IsPackable>true</IsPackable>
-    <PackageDescription>piTop module.</PackageDescription>
+    <PackageDescription>pi-top board.</PackageDescription>
```

Comment:
The device is a `pi-top [4]`, and it is not a board. Let's try and come up with a better description for this! 😄

Reply:
Please let us know as we are almost ready for merge:)